test/Driver/multiple_swiftmodule_inputs.swift

```diff
@@ -0,0 +1,100 @@
+// RUN: %swiftc_driver -c -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
```

Comment:
Please don't actually run these tests; just use `-driver-print-jobs` (alias `-###`) to see what they *would* do. That lets us run the tests even in cross-compilation contexts.